### PR TITLE
Formatting and add install plan related documentation

### PR DESCRIPTION
**Pull request by harishsurf**
This change fixes formatting and adds install plan related documentation. 
Note: I couldn't find much info from existing docs on `InstallPlan`. It definitely needs to be updated

### CODE REVIEW

File: content/en/docs/Tasks/Troubleshooting/install-plan.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+
+The primary way an InstallPlan can fail is by not resolving the resources needed to install a CSV.
+
+```yaml
+apiVersion: app.coreos.com/v1alpha1
+kind: InstallPlan
+metadata:
+  namespace: ci-olm-pr-188-gc-csvs
```
[the remark this answers is not in the document]
Comment (harishsurf) [reply content/en/docs/Tasks/Troubleshooting/install-plan.md:17]: Do you mean the `namespace` field?

File: content/en/docs/Tasks/Troubleshooting/install-plan.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+title: "InstallPlan"
+linkTitle: "InstallPlan"
+date: 2020-03-25
+weight: 4
+description: >
+  Tips and tricks related to troubleshooting a `InstallPLan`.
+---
+
+
+The primary way an InstallPlan can fail is by not resolving the resources needed to install a CSV.
```
[the remark this answers is not in the document]
Comment (anik120) [reply content/en/docs/Tasks/Troubleshooting/install-plan.md:11]: @njhale seems like these should be in the troubleshooting doc. The InstallPlan doc can deal with what the CRD is, and can link to a section in the troubleshooting doc that has a list of expected behaviors on failure and steps to resolve them.

File: content/en/docs/Tasks/Troubleshooting/install-plan.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (njhale) [reply content/en/docs/Tasks/Troubleshooting/install-plan.md:17]: yes

**Comment by anik120** [comment]
Closing due to inactivity

**Pull request closed without merging**